tests/scripts/analyze_outcomes.py

```diff
@@ -87,20 +122,54 @@ def read_outcome_file(outcome_file):
                 outcomes[key].failures.append(setup)
     return outcomes
 
-def analyze_outcome_file(outcome_file):
-    """Analyze the given outcome file."""
+def do_analyze_coverage(outcome_file):
+    """Perform coverage analyze."""
+    outcomes = read_outcome_file(outcome_file)
+    results = analyze_outcomes(outcomes)
+    return results.error_count == 0
+
+def do_analyze_driver_vs_reference(outcome_file, components, ignored_tests):
+    """Perform driver vs reference analyze."""
+    # We need exactly 2 components to analyze (first driver and second reference)
+    if(len(components) != 2 or "accel" not in components[0] or "reference" not in components[1]):
+        print('Error: Wrong component list. Exactly 2 components are required (driver,reference). ')
+        return False
     outcomes = read_outcome_file(outcome_file)
-    return analyze_outcomes(outcomes)
+    return analyze_driver_vs_reference(outcomes, components, ignored_tests)
 
 def main():
     try:
         parser = argparse.ArgumentParser(description=__doc__)
         parser.add_argument('outcomes', metavar='OUTCOMES.CSV',
                             help='Outcome file to analyze')
+        parser.add_argument('--task', default='analyze_coverage',
+                            help='Analyze to be done: analyze_coverage or '
+                            'analyze_driver_vs_reference')
+        parser.add_argument('--components',
```

Comment:
I don't think we need `--components` and `--ignore` options. I think it would be better to have more tasks `coverage` and  `driver_vs_reference_hashes` now, then in the near future `driver_vs_reference_ciphers`, and both `driver_vs_reference_xxx` will look up the component name and ignore list in a table that's included in the script, and call the appropriate function with those. (Also means we need less parsing & validation of inputs.)

That way, when we make progress in driver support, we edit both components in `all.sh` and the data in this scripts, and everything works, without the need to make a PR to mbedtls-test to change this script's command line options.